### PR TITLE
Create aayush.json

### PR DESCRIPTION
**Pull request by OpAayush**

### CODE REVIEW

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]
Please match the format: https://docs.is-a-good.dev/format

**Comment by OpAayush** [comment]
Now is it correct?

**Review by WilliamDavidHarrison — APPROVED** [review]

**Comment by Tweak4141** [comment]
> Now is it correct? Btw are text records supported?

As I said earlier, your txt records should be with your main domain file. This file.
You also have invalid json in your file, hence the check could not pass.

**Comment by OpAayush** [comment]
please can you give the code

**Comment by WilliamDavidHarrison** [comment]
> please can you give the code

Check the docs.

**Comment by wclarkey** [comment]
> > please can you give the code
> 
> Check the docs.

https://docs.is-a-good.dev

**Comment by OpAayush** [comment]
now is it ok?

**Comment by wclarkey** [comment]
Your format is still incorrect. Read this section of the docs https://docs.is-a-good.dev/format#json-example

File: sub-logs/aayush.json [hunk 1]
```diff
@@ -1,15 +1,14 @@
 {
-  "repo": "https://opaayush.pages.dev",
+  "repo": "opayush.pages.dev",
```
Comment (WilliamDavidHarrison) [inline sub-logs/aayush.json:2]: Not a valid repo, please remove this field.

File: sub-logs/aayush.json [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
     "username": "OpAayush",
     "email": "aayushmagrawal@gmail.com"
-    "discord": "Aayu5h#1737"
-   },
-   "target": {
-    "CNAME": {
-       "name": "Aayush", 
+  },
+  "target": {
+    "record-type": {
```
Comment (WilliamDavidHarrison) [inline sub-logs/aayush.json:8]: Change record-type to CNAME.

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Pull request closed without merging**